spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -199,6 +203,20 @@ public DistributionMode copyOnWriteDeleteDistributionMode() {
     }
   }
 
+  public DistributionMode copyOnWriteUpdateDistributionMode() {
+    String updateModeName = confParser.stringConf()
+        .option(SparkWriteOptions.DISTRIBUTION_MODE)
+        .tableProperty(TableProperties.UPDATE_DISTRIBUTION_MODE)
+        .parseOptional();
+
+    if (updateModeName != null) {
+      DistributionMode updateMode = DistributionMode.fromName(updateModeName);
+      return adjustWriteDistributionMode(updateMode);
```

Comment:
As with the copy-on-write delete case, I think I'd prefer to preserve the existing file boundaries and row order if the table is unpartitioned and unsorted. I would also do that as the default, if neither the write order or the update order is specified for unpartitioned tables.

Reply:
Let me think.